R/type.R

```diff
@@ -99,8 +99,7 @@ xportr_type <- function(.df,
 
   ## Common section to detect domain from argument or pipes
 
-  df_arg <- tryCatch(as_name(enexpr(.df)), error = function(err) NULL)
-  domain <- get_domain(.df, df_arg, domain)
+  domain <- get_domain(.df, domain)
```

Comment:
What do you think about a new function say `update_domain` that does both these steps together:
```r
domain <- get_domain(.df, domain)
if (!is.null(domain)) attr(.df, "_xportr.df_arg_") <- domain
```
So, we can simply do a `domain <- update_domain(.df, domain)`

Reply:
I don't think will work because the '.df' object isn't retaining the attr because it can't retain it in the function.